.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,12 @@
+## Contributing
+
+Please join us on [chat](https://chat.awesome-selfhosted.com) to clarify any questions.
+
+ * The **syntax** for links is ``[Name](http://homepage/) - Short description. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code)) `License` `Language` ``. All projects should have their **license** and main server-side **language**/platform/requirement listed, and a description at most 250 characters long.
```

Comment:
[Name](http://homepage/) - Short description. ([Demo](http://url.to/demo), [Example](http://url.of/example), [Source Code](http://url.of/source/code)) `License` `Language`